x/storage/types/keys.go

```diff
@@ -156,3 +156,7 @@ func GetInternalBucketInfoKey(bucketID math.Uint) []byte {
 	var seq sequence.Sequence[math.Uint]
 	return append(InternalBucketInfoPrefix, seq.EncodeSequence(bucketID)...)
 }
+
+func GetResourceTagKey(resourceType string, resourceID math.Uint) []byte {
+	return append([]byte(resourceType), resourceID.Bytes()...)
```

Comment:
you must have a prefix, otherwise storage override will happen

Reply:
Done